File: app/src/main/java/com/llj/architecturedemo/view/MainContractView.java
package com.llj.architecturedemo.view;

import com.llj.architecturedemo.db.entity.MobileEntity;
import com.llj.lib.base.mvp.IView;

/**
 * ArchitectureDemo
 * describe:
 * author liulj
 * date 2018/6/4
 */
public interface MainContractView extends IView {

    void toast(MobileEntity mobile);
}

File: app/src/p_app/main/java/com/llj/architecturedemo/view/MainContractView.kt
package com.llj.architecturedemo.view

import com.llj.architecturedemo.db.entity.MobileEntity
import com.llj.lib.base.mvp.IView

/**
 * ArchitectureDemo
 * describe:
 * author liulj
 * date 2018/6/4
 */
interface MainContractView : IView {

    fun toast(mobile: MobileEntity?)
}
